google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/it/SampleRowsIT.java

```diff
@@ -61,4 +67,50 @@ public void test() throws InterruptedException, ExecutionException, TimeoutExcep
     assertThat(results).isNotEmpty();
     assertThat(results.get(results.size() - 1).getOffsetBytes()).isGreaterThan(0L);
   }
+
+  @Test
+  public void testOnAuthorizedView()
+      throws InterruptedException, ExecutionException, TimeoutException {
+    assume()
+        .withMessage("AuthorizedView is not supported on Emulator")
+        .that(testEnvRule.env())
+        .isNotInstanceOf(EmulatorEnv.class);
+
+    AuthorizedView testAuthorizedView = createTestAuthorizedView(testEnvRule);
+
+    BigtableDataClient client = testEnvRule.env().getDataClient();
+    String rowPrefix = AUTHORIZED_VIEW_ROW_PREFIX + UUID.randomUUID();
+    String rowPrefixOutsideAuthorizedView = UUID.randomUUID() + "-outside-authorized-view";
+
+    // Create some data so that sample row keys has something to show
+    List<ApiFuture<?>> futures = Lists.newArrayList();
+    for (int i = 0; i < 10; i++) {
+      ApiFuture<Void> future =
+          client.mutateRowAsync(
+              RowMutation.create(testEnvRule.env().getTableId(), rowPrefix + "-" + i)
+                  .setCell(
+                      testEnvRule.env().getFamilyId(), AUTHORIZED_VIEW_COLUMN_QUALIFIER, "value"));
+      futures.add(future);
+      ApiFuture<Void> futureOutsideAuthorizedView =
+          client.mutateRowAsync(
+              RowMutation.create(
+                      testEnvRule.env().getTableId(), rowPrefixOutsideAuthorizedView + "-" + i)
+                  .setCell(
+                      testEnvRule.env().getFamilyId(), AUTHORIZED_VIEW_COLUMN_QUALIFIER, "value"));
+      futures.add(futureOutsideAuthorizedView);
+    }
+    ApiFutures.allAsList(futures).get(1, TimeUnit.MINUTES);
+
+    ApiFuture<List<KeyOffset>> future = client.sampleRowKeysAsync(testEnvRule.env().getTableId());
```

Comment:
this is doing sample row keys on the table instead of the authorized view? 

we'll need to create a table with pre splits and some splits have authorized view rows and some don't